examples/usb/doc/rst/usb.rst

```diff
@@ -161,7 +161,8 @@ resource, setting LEDs or reading the buttons.
 
 Please see the source code listing app_listing_ for more details.
 
-|appendix|
+.. TODO: Check how the substitution below worked and update it.
```

Comment:
I have seen a few ToDo's.  They need to be resolved before merging or have a story created that includes them.

Reply:
[LSM-120](https://xmosjira.atlassian.net/browse/LSM-120) covers rewriting the documentation.
The readme files should be updated there.